auth-lib/src/main/AndroidManifest.xml

```diff
@@ -44,7 +44,6 @@
         </activity>
 
         <activity
-            android:launchMode="singleInstance"
```

Comment:
It was added in 8a97adf , I assumed we needed this so that we don't launch multiple instances of this activity.